app/resources/en/translation/declaration.json

```diff
@@ -1,6 +1,6 @@
 {
   "title": "Check the legal statement and make your declaration",
-  "intro": "This statement is based on the information you&rsquo;ve given in your application. It will be stored as a public record.",
+  "intro": "This statement is based on the information you've given in your application. It will be stored as a public record.",
```

Comment:
We should be using `&rsquo;` here to keep consistency across the app

Reply:
I dont think we should have escaped chars in 'content' if we can avoid it

Reply:
We've done this throughout the app because the decision was taken a long time ago by UX that having proper 66 and 99's was looked better than having `'`.

If we're going to decide to change that then we should change them throughout the app rather than just in one place.